dependencies.yaml

```diff
@@ -641,7 +641,7 @@ dependencies:
               arch: x86_64
             packages:
               # Currently, CUDA + aarch64 builds of pytorch do not exist on conda-forge.
-              - pytorch>=2.1.0
+              - pytorch
```

Comment:
For testing this PR, I think you can comment out pytorch and related dependencies (tokenizers, transformers) - the tests using them are optional, iirc.